### PR TITLE
Add `SetupGreetingTest`

### PR DESCRIPTION
**Pull request by liweinan**

### CODE REVIEW

**Comment by liweinan** [comment]
@ronsigal I created this class to automate the process to copy the test class file and setup the pom deps in bridge project.

The only problem is that the `GreetingTest` in the `resources` dir is not copied to the target bridge project. I guess this copy action needs to be added in `gRPCtoJakartaREST-archetype` right?

File: grpc-bridge-example/src/main/java/dev/resteasy/example/grpc/greet/SetupGreetingTest.java
Comment (jamezp) [inline grpc-bridge-example/src/main/java/dev/resteasy/example/grpc/greet/SetupGreetingTest.java:1]: This needs a copyright header.

**Review by jamezp — COMMENTED** [review]
I'm not too sure about this. IMO if we need a separate class to configure the project, the setup is too complicated and we should look at fixing that.

File: grpc-bridge-example/src/main/java/dev/resteasy/example/grpc/greet/SetupGreetingTest.java
Comment (liweinan) [reply grpc-bridge-example/src/main/java/dev/resteasy/example/grpc/greet/SetupGreetingTest.java:1]: yes sir!

**Comment by liweinan** [comment]
> I'm not too sure about this. IMO if we need a separate class to configure the project, the setup is too complicated and we should look at fixing that.

Currently the bridge project generation process is controlled by the [gRPCtoJakartaREST-archetype](https://github.com/resteasy/gRPCtoJakartaREST-archetype) project, and the detail steps in the generation process determines that such kind of `GreetingTest` must be put into position *after* all the generated sources are ready, or the `GreetingTest` that uses the generated classes will cause the build process failed.

+1 that we simplify this process in future.

**Comment by liweinan** [comment]
In addition I have created a discussion here related with this PR: https://github.com/resteasy/gRPCtoJakartaREST-archetype/discussions/9

**Comment by jamezp** [comment]
If you have to write code to update a project before you can use the project, something is wrong and IMO that needs to be addressed. I don't think we should patch this and hope we fix it, I think we need to work on the integration code and make it more user friendly. 

Just my opinion :)

**Comment by liweinan** [comment]
@jamezp Okay!

**Comment by jamezp** [comment]
I'm going to close this for now as it's got conflicts and it seems we need to think about it more.

**Pull request closed without merging**